downstream/titles/eda/eda-getting-started-guide/master.adoc

```diff
@@ -0,0 +1,29 @@
+:imagesdir: images
+:numbered:
+:toclevels: 1
+:context: eda-getting-started
+
+:experimental:
+
+include::attributes/attributes.adoc[]
+
+
+// Book Title
+= Getting Started with {EDAname} Guide
+
+Thank you for your interest in {EDAname}. {EDAname} is a new way to enhance and expand automation. It helps teams automate decision-making and improve IT speed and agility .
```

Comment:
@jself-sudoku There is a blank space between agility and the period ending the sentence. I didn't see it until I formatted the doc locally to verify a clean build.